.github/workflows/code-testing.yml

```
@@ -158,9 +158,12 @@ jobs:
         with:
           python-version: "3.11"
       - name: Install dependencies
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)